appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java

```diff
@@ -831,7 +831,7 @@ private String getBuildServerUrlStr(String userName, String userId,
 
   private String getCurrentHost() {
     if (Server.isProductionServer()) {
-      if (appengineHost.get()=="") {
+      if ("".equals(appengineHost.get())) {
```

Comment:
I believe that this will always return a not-null value, so you could use `isEmpty()` instead of checking equality to `""`.